src/main/java/com/conveyal/gtfs/loader/ReferenceTracker.java

```diff
@@ -160,18 +160,22 @@
         // Work through each field that has been assigned a conditional requirement.
         for (Map.Entry<Field, ConditionalRequirement[]> entry : fieldsToCheck.entrySet()) {
             Field referenceField = entry.getKey();
-            ConditionalRequirement[] conditionalRequirements = entry.getValue();
             // Work through each field's conditional requirements.
-            for (ConditionalRequirement check : conditionalRequirements) {
+            for (ConditionalRequirement check : entry.getValue()) {
```

Comment:
I'm sorry, my suggestion makes the new code hard to read, could you change back?...

Reply:
No worries, restored.